### DISCUSSION TITLE
Force demand supply

### DISCUSSION THREAD

**Discussion by matteocarcano** [frePPLe/frepple#674] [Q&A]
Hi,
assuming there are more manufacturing order  confirmed on the same item, is there any possibility to force demand's supply on one or more specific manufacturing order?

thank you

**Comment by jdetaeye** [comment]
Check out the "make-to-order" feature: https://frepple.com/docs/current/user-guide/examples/buffer/make-to-order.php

Using the item.type field and the batch field on sales orders, buffer, manufacturing orders, purchase orders and distribution orders you can make your supply specific for a certain group of demands.   You can do this for the complete supply path (ie full make-to-order), or a part of your supply path (eg assemble-to-order, serialized production).

**Comment by jdetaeye** [comment]
Check out the "make-to-order" feature: https://frepple.com/docs/current/user-guide/examples/buffer/make-to-order.php

Using the item.type field and the batch field on sales orders, buffer, manufacturing orders, purchase orders and distribution orders you can make your supply specific for a certain group of demands.   You can do this for the complete supply path (ie full make-to-order), or a part of your supply path (eg assemble-to-order, serialized production).

**Comment by jdetaeye** [comment]
Check out the "make-to-order" feature: https://frepple.com/docs/current/user-guide/examples/buffer/make-to-order.php

Using the item.type field and the batch field on sales orders, buffer, manufacturing orders, purchase orders and distribution orders you can make your supply specific for a certain group of demands.   You can do this for the complete supply path (ie full make-to-order), or a part of your supply path (eg assemble-to-order, serialized production).
